### DISCUSSION TITLE
Center part of the conveyor dosen't move

### DISCUSSION THREAD

**Discussion by tiagocunico** [Open-Industry-Project/Open-Industry-Project#76] [Q&A] [answered]
Sometimes when I start the project, the center part of the conveyor doesn't move, just the sides. The box doesn't spawn either. What can I do?

![ErroEsteira](https://github.com/user-attachments/assets/4baaf2de-1108-4a01-b24a-3049a1b2a149)

Sorry if it's a dumb question, first time using Godot.

**Comment by ryevdokimov** [comment, marked as answer]
This will be fixed in the next version. 

For now, closing and reopening the scene should fix the problem: 

![image](https://github.com/user-attachments/assets/ed985cd5-2597-48d5-a7ec-c8db3551810c)
